manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.OpenJDK.11
+PackageVersion: 11.0.14.9
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-11.0.14.9.1-windows-x64.msi#winget
```

Comment:
```suggestion
  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/70109184-0b6e-430d-8e8e-35e8e65fe8af/e83bc9e615401db2e3e0b3c5c8fb31a3/microsoft-jdk-11.0.14.9.1-windows-x64.msi

```

Reply:
Leaving, we use aka.ms/ links to verify usage.

Reply:
> Leaving, we use aka.ms/ links to verify usage.

Previous versions use the direct links.